src/Analysis/GWASV2/Components/Diagrams/CohortsOverlapDiagram/Simple3SetsOverlapTextVersion.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CohortsOverlapTextVersion = ({ title, eulerArgs }) => {
```

Comment:
maybe rename this to match the file name?

Reply:
Good catch, this has been updated.